src/Gelf/Logger.php

```diff
@@ -181,7 +181,7 @@ private static function interpolate($message, array $context)
         // build a replacement array with braces around the context keys
         $replace = array();
         foreach ($context as $key => $val) {
-            $replace['{' . $key . '}'] = $val;
+            $replace['{' . $key . '}'] = is_string($val) ? $val : 'Array';
```

Comment:
@bzikarsky it sure is, but I think it'd be best if I extract it out of monolog so you can depend on it instead, it kinda grew into its own thing :)